embedded/ActuatorBoard/readme.md

```diff
@@ -1,27 +0,0 @@
-**Actuator Board**
```

Comment:
Electrical documentation is being kept on Confluence, the FW is being kept along side the PCB project in SVN.